arangodb-net-standard.Test/GraphApi/Models/PutVertexMockModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace ArangoDBNetStandardTest.GraphApi.Models
+{
+    public class PutVertexMockModel
+    {
```

Comment:
What about Old and New properties, I think we have included those elsewhere, we should include them here too.

Reply:
Oh, sorry! I was looking at the wrong model, it seems you did handle Old and New in the response model.